app/models/rate_management/pricelist_state.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+# == Schema Information
+#
+# Table name: ratemanagement.pricelist_states
+#
+#  id   :bigint(8)        not null, primary key
+#  name :string           not null
+#
+module RateManagement
+  class PricelistState < ApplicationRecord
+    self.table_name = 'ratemanagement.pricelist_states'
+
+    module CONST
+      STATE_ID_APPLIED = 10
+      STATE_ID_DIALPEERS_DETECTED = 20
+      STATE_ID_NEW = 30
+
+      freeze
+    end
```

Comment:
let's get rid of `RateManagement::PricelistState` model and database table.  
1. remove `ratemanagement.pricelist_states` database table from migration
2. `delete class RateManagement::PricelistState` and it's association in `RateManagement::Pricelist`
3. move constants `STATE_ID_*` to `RateManagement::Pricelist::CONST`
4. add `STATE_IDS = { ... }.freeze` to `RateManagement::Pricelist::CONST` and use it in admin UI
5. validate `state_id` column via inclusion validator on `RateManagement::Pricelist`